carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/ftp/FtpUtils.java

```diff
@@ -73,7 +73,11 @@ private static void upload(String ftpHost, int port, String user, String passwor
 			LOGGER.debug("User has been successfully logged in.");
 			ftp.setFileType(FTP.BINARY_FILE_TYPE);
 			try {
-			    ftp.enterLocalPassiveMode();
```

Comment:
could you please return recommended enterLocalPassiveMode and keep debug log messages